### DISCUSSION TITLE
Windows 11 installation freezes

### DISCUSSION THREAD

**Discussion by stevofusek** [utmapp/UTM#3185] [UTM Help (macOS)]
Hi, I have a problem with the installation of Windows 11 on the M1 Macbook Pro 16GB version. The installation freezes randomly and I can't even get into the desktop screen.
macOS ver.: 11.6
UTM ver.: 2.2.4
Windows ver.: 22454

**Comment by Francesco146** [comment]
https://github.com/utmapp/UTM/issues/3141#issue-1008371318

here same problem, but they say they fixed it with the latest Windows build.. let us know if you solve it

**Comment by stevofusek** [comment]
No I didn't, even tried the latest version and still no luck

**Comment by dude19me** [comment]
im having the same problem

**Comment by DzigaVertoff** [comment]
Same problem here
macOS 11.6
UTM 2.2.4 (36)
Windows ARM64 version 22454

**Comment by renatogdn** [comment]
Same thing here.

I went through all the steps but Microsoft is only making available Windows 11 now instead of windows 10 Developer's preview.

So, installation starts fine but freezes at some point and won't conclude.
I'm using Windows 11 build #22454.

If anyone has a work around, please share!
Thanks in advance...

**Comment by muratdk23** [comment]
> Same thing here.
> 
> I went through all the steps but Microsoft is only making available Windows 11 now instead of windows 10 Developer's preview.
> 
> So, installation starts fine but freezes at some point and won't conclude. I'm using Windows 11 build #22454.
> 
> If anyone has a work around, please share! Thanks in advance...

Works in Vmware)

**Comment by FraFraFra-LongD** [comment]
> Works in Vmware

VMWare is not available on M1 MacBooks, is it?

**Comment by muratdk23** [comment]
> > Works in Vmware
> 
> VMWare is not available on M1 MacBooks, is it?

vmware released public beta for arm.

**Comment by DzigaVertoff** [comment]
> > > Works in Vmware
> 
> > 
> 
> > VMWare is not available on M1 MacBooks, is it?
> 
> 
> 
> vmware released public beta for arm. 

Can you link it? I found closed beta only

**Comment by muratdk23** [comment]
> > > > Works in Vmware
> > 
> > 
> > > 
> > 
> > 
> > > VMWare is not available on M1 MacBooks, is it?
> > 
> > 
> > vmware released public beta for arm.
> 
> Can you link it? I found closed beta only

Search in google "VMware Fusion Public Tech Preview 21H1".

**Comment by muratdk23** [comment]
Worked for me.
1. Install windows 10 for arm build 21390.
2. Activate beta builds through offlin insider enroll. 
3. Install windows 11 build 22000.160 through windows update.

**Comment by DzigaVertoff** [comment]
> Worked for me.
> 
>     1. Install windows 10 for arm build 21390.
> 
>     2. Activate beta builds through offlin insider enroll.
> 
>     3. Install windows 11 build 22000.160 through windows update.

Where can I download windows 10 arm build 21390?

**Comment by muratdk23** [comment]
> > Worked for me.
> > ```
> > 1. Install windows 10 for arm build 21390.
> > 
> > 2. Activate beta builds through offlin insider enroll.
> > 
> > 3. Install windows 11 build 22000.160 through windows update.
> > ```
> 
> Where can I download windows 10 arm build 21390?

Uup dump.

**Comment by dooitze** [comment]
Is there perhaps a .utm profile which I can download with Windows 10/11 already installed for Apple Silicon? I have tried it multiple times already, also in the ways above but I consistently get an error (0x800701B1).

**Comment by DzigaVertoff** [comment]
> Worked for me.
> 
>     1. Install windows 10 for arm build 21390.
> 
>     2. Activate beta builds through offlin insider enroll.
> 
>     3. Install windows 11 build 22000.160 through windows update.

Tried to install windows 10 for arm build 21390. But there is this message when Windows installation starts: "A media driver your computer needs is missing, ecc."

**Comment by lookoutldz** [comment]
> > > Worked for me.
> > > ```
> > > 1. Install windows 10 for arm build 21390.
> > > 
> > > 2. Activate beta builds through offlin insider enroll.
> > > 
> > > 3. Install windows 11 build 22000.160 through windows update.
> > > ```
> > 
> > 
> > Where can I download windows 10 arm build 21390?
> 
> Uup dump.

Can UUD dump  download .VHDX file? I just see that UTM using .VHDX file to create new VM but not .ISO file.
Here is the guide:
https://mac.getutm.app/gallery/windows-10-arm

**Comment by conath** [comment]
Moving this to Discussions since not UTM issue.

**Comment by dooitze** [comment]
So what I ended up doing is using VMware Fusion Tech Preview to configure the .VHDX file and afterwards I imported it into the UTM virtual machine. I then installed the Spice Tools which enabled networking and varying screen resolution. It now runs relatively stable on my MacBook Air M1.
